gno.land/pkg/sdk/vm/common_test.go

```diff
@@ -47,7 +48,9 @@ func _setupTestEnv(cacheStdlibs bool) testEnv {
 	ctx := sdk.NewContext(sdk.RunTxModeDeliver, ms, &bft.Header{ChainID: "test-chain-id"}, log.NewNoopLogger())
 	acck := authm.NewAccountKeeper(iavlCapKey, std.ProtoBaseAccount)
 	bank := bankm.NewBankKeeper(acck)
-	vmk := NewVMKeeper(baseCapKey, iavlCapKey, acck, bank, 100_000_000)
+	prmk := paramsm.NewParamsKeeper(iavlCapKey, "params")
+	maxCycles := int64(100_000_000) // XXX: use x/params for 100_000_000
```

Comment:
will this be removed here?

Reply:
Yup -> https://github.com/gnolang/gno/pull/2993